doc/lua_api.md

```diff
@@ -4081,30 +4081,36 @@ Two functions are provided to translate strings: `minetest.translate` and
   the given `textdomain` for disambiguation. The textdomain must match the
   textdomain specified in the translation file in order to get the string
   translated. This can be used so that a string is translated differently in
-  different contexts.
+  different contexts.  
```

Comment:
Can we please not use markdown's double space to insert line breaks?

Many editors trim them automatically (which is usually a good thing), and even our .editorconfig suggests this: https://github.com/minetest/minetest/blob/5727d74d37597927f3b67cfa5a8cefc1eee4fc85/.editorconfig#L9

Also, if you want to start a new paragraph, you should insert an empty line instead. Line breaks are often not even visible in the output, because the upper line goes completely to the right end of the page.

Reply:
Alright, removed them again,